src/components/admonition/examples/admonition-composite.tsx

```diff
@@ -0,0 +1,68 @@
+import { Component, h, Prop, State } from '@stencil/core';
+
+/**
+ * Composite
+ *
+ * Show different types
+ */
+@Component({
+    tag: 'limel-example-admonition-composite',
+    shadow: true,
+})
+export class AdmonitionCompositeExample {
+    @Prop()
+    public schema: any;
+
+    @State()
+    private props: any = {
+        title: '',
+        description: 'This is my very nice [type]',
+        type: 'tip',
+    };
+
+    public componentWillLoad() {
+        const properties = { ...this.schema.properties };
+
+        this.schema = {
+            ...this.schema,
+            properties: properties,
+            lime: {
+                layout: {
+                    type: 'grid',
+                },
+            },
+        };
+    }
+
+    public render() {
+        const description = this.props?.description?.replace(
+            '[type]',
+            this.props?.type ?? ''
+        );
+
+        return [
+            <limel-admonition
+                {...{ ...this.props, description: description }}
```

Comment:
Haha! This double spread made my brain put a knot in itself, before I realised what's going on! 😂

Doesn't the following work? I think it's a little clearer to read 🙂

```suggestion
                { ...this.props }
                description={ description }
```